docs/pages/configuration/service.mdx

```diff
@@ -34,6 +34,9 @@ The `port` option is mandatory and expect an integer with a port number. The ser
 ### `containerPort`
 The `containerPort` option is optional and tells the service to forward the traffic from the service port (defined in `port`) to a different `containerPort` of a pod.
 
+### `name`
+The `name` option is optional and allows you to customise the name of the port. By default ports will be named "port-0", "port-1" etc.
```

Comment:
Looks great. Thank you for contributing! I'd only have 1 minor change request: please change BE "customize" to AE "customize" for consistency.

Reply:
No problem. Updated now to make it less Britsh 😋